.gitignore

```diff
@@ -9,3 +9,4 @@ npm-debug.*
 web-build/
 web-report/
 prototype/
+yarn-error.log
```

Comment:
Oltre che ignorarlo in git tocca rimuoverlo dalla PR sennò lo committi comunque nel repo 🙈